ext/RMagick/rmfill.c

```diff
@@ -361,14 +361,14 @@ v_diagonal_fill(
     green_step = ((MagickRealType)stop_color->green - (MagickRealType)start_color->green) / steps;
     blue_step =  ((MagickRealType)stop_color->blue  - (MagickRealType)start_color->blue)  / steps;
 
-    for (y = 0; y < image->rows; y++)
+    for (y = 0; y < (ssize_t) image->rows; y++)
```

Comment:
unnecessary type cast.
```suggestion
    for (y = 0; y < image->rows; y++)
```